deployment/roles/ui/tasks/servinstall.yml

```diff
@@ -0,0 +1,17 @@
+# This role creates a tokenbridge_ui service which is designed to manage docker-compose ui deployment.
+# /etc/init.d/tokenbridge_ui start, status, stop, restart - does what the services usually do in such cases.
+# /etc/init.d/tokenbridge_ui rebuild - builds a new ui deployment from scratch.
+---
+- name: "Set the service"
+  template:
+    src: tokenbridge_ui.j2
+    dest: "/etc/init.d/tokenbridge_ui"
```

Comment:
although there is not strict naming convention for names of the linux services I did a following experiment:
```shell
$ ls -1 /etc/init.d/ | grep '_' | wc -l
0
$ ls -1 /etc/init.d/ | grep '-' | wc -l
20
```
So, as you can see there is no services containing the underscore in the name. Do you have any objections to change the name as so it contain the hyphen instead of the underscore?

Reply:
Please review https://github.com/poanetwork/tokenbridge/pull/126/commits/c11012ba3935ab147954d91863898abeba546bcc
I have also modified documentation.